build/jobs/provision-deploy.yml

```diff
@@ -54,7 +54,7 @@ jobs:
         $templateParameters = @{
             fhirVersion = "${{ parameters.version }}"
             appServicePlanName = "${{ parameters.appServicePlanName }}"
-            appServicePlanSku = "B2"
+            appServicePlanSku = "B3"
```

Comment:
Was the cause of the test errors related to low memory? Did this improve the PR process?

Reply:
The PR did pass without issues after this change, so I do think it was the cause.  I believe @rbans96 was seeing the same issue and going to try the same fix.  We can check with her to confirm.